src/Evolution/EventsAndDenseTriggers/EventsAndDenseTriggers.hpp

```diff
@@ -31,6 +31,16 @@ struct TimeStepId;
 /// \endcond
 
 namespace evolution {
+namespace Tags {
+/*!
+ * \ingroup EventsAndTriggersGroup
+ * \brief Previous time at which the trigger activated.
+ */
```

Comment:
This tag acts a little strangely from a user's perspective, so maybe emphasize that it is for use in an Event to query information about the trigger associated with that event.